ultimarc/devices/mini_pac.py

```diff
@@ -103,9 +103,15 @@ def to_json_str(self, pac_struct):
         """ Converts a PacStruct to a json object """
         json_obj = {'schemaVersion': 2.0, 'resourceType': 'mini-pac-pins', 'deviceClass': self.class_id}
 
-        # debounce
-        # TODO - fully implement debounce behavior
-        json_obj['debounce'] = 'standard'
+        # header configuration
+        header = PacConfigUnion()
+        header.asByte = pac_struct.header.byte_4
+        debounce = get_ipac_series_debounce_key(header.config.debounce)
+        if debounce is not None:
+            json_obj['debounce'] = debounce
+        else:
+            json_obj['debounce'] = 'standard'
+            _logger.info(_(f'"{header.config.debounce}" debounce value is not a valid value'))
```

Comment:
It seems to me that it would be better to move the 'debounce' if, else and logging logic (110:114) to the `get_ipac_series_debounce_key()` function.  So that any call to 'get_ipac_series_debounce_key()' will always return a good value.  If that is not what you want to do, let me know.

Reply:
Yes it would be better to move that into get_ipac_series_debounce_key method.  This will be used with a number of other devices.